ui/desktop/electron-app/src/cli/path.js

```diff
@@ -20,6 +21,16 @@ module.exports = {
       : path.resolve(process.resourcesPath, 'cli', binaryName);
     const isBuiltInCli = existsSync(builtInCliPath);
```

Comment:
I think we can check for `isBuiltInCli` to return `builtInCliPath` immediately here so that way we don't have to do two ternary statements after